core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -600,11 +577,8 @@ private static Sep31InfoResponse sep31InfoResponseFromAssetInfoList(List<AssetIn
         AssetResponse assetResponse = new AssetResponse();
         assetResponse.setQuotesSupported(isQuotesSupported);
         assetResponse.setQuotesRequired(isQuotesRequired);
-        assetResponse.setFeeFixed(assetInfo.getSep31().getReceive().getFeeFixed());
-        assetResponse.setFeePercent(assetInfo.getSep31().getReceive().getFeePercent());
         assetResponse.setMinAmount(assetInfo.getSep31().getReceive().getMinAmount());
         assetResponse.setMaxAmount(assetInfo.getSep31().getReceive().getMaxAmount());
-        assetResponse.setFields(assetInfo.getSep31().getFields());
         response.getReceive().put(assetInfo.getCode(), assetResponse);
```

Comment:
Can we directly use `Sep31Info` here instead of mapping it to `assetResponse`?  They contain the same info essentially.

Reply:
Although it saves coding, it is usually safer to separate the concern of the /info response and the assets configuration.

Making them the same can unintentionally break the SEP-31 `info` response when we change the implementation.

Reply:
Per our discussion, I will create another PR to add `sep31.receive.methods`